### PR TITLE
Add README.zh-CN.md

### LINKED ISSUE #176: The 'ultralytics/ultralytics' repository doesn't contain the 'README.zh-CN.md' path in 'main'.
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

The 'ultralytics/ultralytics' repository doesn't contain the 'README.zh-CN.md' path in 'main'.  

### Additional

_No response_

### PR DESCRIPTION
**Pull request by RangeKing**
Add `README.zh-CN.md`

## 🛠️ PR Summary

<sub>Made with ❤️ by [Ultralytics Actions](https://github.com/ultralytics/actions)<sub>

### 🌟 Summary
Introduction of a Chinese README for Ultralytics' GitHub repository.

### 📊 Key Changes
- Added a Chinese (Simplified) version of the README file (`README.zh-CN.md`), making the project more accessible to Chinese-speaking developers and users.
- Included software installation and usage examples, integrating YOLO tasks such as detection, classification, segmentation, and model export.
- Provided information on model performance metrics for various versions of YOLOv8 models, including YOLOv8n, YOLOv8s, YOLOv8m, YOLOv8l, and YOLOv8x in different configurations (standard, segmentation, classification).
- Added badges for continuous integration, citations, Docker pulls, and links to run the Ultralytics models in different cloud platforms including Gradient, Colab, and Kaggle.

### 🎯 Purpose & Impact
- 🗺️ **Enhanced Accessibility**: By localizing documentation, the changes will cater to a wider audience, specifically targeting Chinese developers and users, which can lead to increased adoption and contributions from this demographic.
- 🤝 **Community Engagement**: Having a Chinese version of the README fosters a more inclusive community and encourages contributions from those who are more comfortable with Chinese.
- 📈 **Increased Usability**: The concise examples and commands provided enable new users to get started more easily, reducing entry barriers for using Ultralytics products.
- 📚 **Educational Resources**: The inclusion of various links to educational platforms allows users to learn and interact with Ultralytics products through their preferred learning environments.

### CODE REVIEW

**Comment by RangeKing** [comment]
I have read the CLA Document and I sign the CLA

**Comment by glenn-jocher** [comment]
@RangeKing wow that was fast. Thanks!

**Comment by RangeKing** [comment]
@glenn-jocher my pleasure.

File: README.zh-CN.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+    <br>
+    <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a>
+    <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a>
+    <a href="https://www.kaggle.com/ultralytics/yolov8"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
+  </div>
+  <br>
+
+[Ultralytics YOLOv8](https://github.com/ultralytics/ultralytics) 是由 [Ultralytics](https://ultralytics.com) 开发的一个前沿 SOTA 模型。它在以前 YOLO 版本的成功基础上，引入了新的功能和改进，进一步提升了性能和灵活性。YOLOv8 基于快速、准确和易于使用的理念设计，使其成为广泛的物体检测、图像分割和图像分类任务的绝佳选择。
+
+如如要申请企业许可证，请填写 [Ultralytics 许可](https://ultralytics.com/license)。
```
Comment (Laughing-q) [inline README.zh-CN.md:23]: 如如 -> 如果

Comment (RangeKing) [reply README.zh-CN.md:23]: Get it. I'll fix all typos after your review.

Comment (Laughing-q) [reply README.zh-CN.md:23]: @RangeKing oh actually that's all I found, the rest of them are good! Thanks!

**Comment by Laughing-q** [comment]
@RangeKing Thanks!
@glenn-jocher Then it's ready to go. :)

**Comment by glenn-jocher** [comment]
@RangeKing @Laughing-q PR merged!

**Pull request merged**